api/src/services/graphql.ts

```diff
@@ -1366,7 +1366,7 @@ export class GraphQLService {
 						userAgent: req?.get('user-agent'),
 						role: null,
 					};
-					const authenticationService = new AuthenticationService({
+					const authenticationService = new BasicAuthenticationService({
```

Comment:
I don't even want to deal with  LDAP authentication for GraphQL. Maybe that can be a V2 project.